### PR TITLE
product type completed; all tests pass

### PR DESCRIPTION
**Pull request by emmettSchneider**
# Description
Product Type included all tests pass.

## Type of change
- [X] New feature (non-breaking change which adds functionality)

# Testing Instructions for Change Made
1. `git fetch --all`
1. `git checkout new-product-type-ES`
1. run Azure Data Studio and -- INSERT INTO ProductType ([Name]) VALUES ('Jewellery');
-- INSERT INTO ProductType ([Name]) VALUES ('Stuffed Toy');
-- INSERT INTO ProductType ([Name]) VALUES ('Collectible');
-- INSERT INTO ProductType ([Name]) VALUES ('Used Book');
1. Run ProductType tests and ensure they pass

# Checklist:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by BrianBNeal — COMMENTED** [review]
👍 good job sir

File: TestBangazonAPI/TestProductTypes.cs [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                Assert.Equal("Toys", newProductType.Name);
+
+
+                var deleteResponse = await client.DeleteAsync($"/productType/{newProductType.Id}");
+                deleteResponse.EnsureSuccessStatusCode();
+                Assert.Equal(HttpStatusCode.NoContent, deleteResponse.StatusCode);
+            }
+        }
+
+        [Fact]
```
Comment (naienko) [inline TestBangazonAPI/TestProductTypes.cs:95]: This test is in here twice! And this one wouldn't work anyway.

**Review by naienko — CHANGES_REQUESTED** [review]
:+1:

File: TestBangazonAPI/TestProductTypes.cs (hunk 1 above)
Comment (BrianBNeal) [reply TestBangazonAPI/TestProductTypes.cs:95]: Not sure what you're referring to.  More specifics?

Comment (naienko) [reply TestBangazonAPI/TestProductTypes.cs:95]: I'm not reading closely enough. Ignore this.

**Review by naienko — APPROVED** [review]

**Pull request merged**